model/src/w3gridmd.F90

```diff
@@ -2415,17 +2415,17 @@ SUBROUTINE W3GRID()
 !
 ! 6.l Read unstructured data 
 ! initialisation of logical related to unstructured grid
-       UGOBCAUTO = .TRUE.
-       UGBCCFL = .TRUE.
-       UGOBCDEPTH= -10.
-       UGOBCOK = .FALSE.
-       UGOBCFILE = 'unset'
-       EXPFSN    = .TRUE.
-       EXPFSPSI  = .FALSE.
-       EXPFSFCT  = .FALSE.
-       IMPFSN    = .FALSE.
-       IMPTOTAL  = .FALSE.
-       EXPTOTAL  = .FALSE. 
+       UGOBCAUTO  = .TRUE.
+       UGBCCFL    = .FALSE.
```

Comment:
Again, I hope this will be changed in a separate PR as this is a very impactful change, but I think we should perhaps also change the value when this variable is initialized for example: 
```
           LOGICAL :: UGBCCFL   = .FALSE.
           LOGICAL :: EXPFSN    = .FALSE.
```

Reply:
Please revert this change, create a new PR or provide me with the information for issue #644 so I can help with this.